packages/components/src/base-field/hook.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * External dependencies
+ */
+import { cx } from 'emotion';
+
+/**
+ * WordPress dependencies
+ */
+import { useMemo } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import { useContextSystem } from '../ui/context';
+import { useControlGroupContext } from '../ui/control-group';
+import { useFlex } from '../flex';
+import * as styles from './styles';
+
+/**
+ * @typedef OwnProps
+ * @property {boolean} [error=false] Renders an error.
+ * @property {boolean} [disabled] Whether the field is disabled.
+ * @property {boolean} [isClickable=false] Renders a `cursor: pointer` on hover.
+ * @property {boolean} [isFocused=false] Renders focus styles.
+ * @property {boolean} [isInline=false] Renders as an inline element (layout).
+ * @property {boolean} [isSubtle=false] Renders a subtle variant.
+ */
+
+/** @typedef {import('../Flex/useFlex').FlexProps & OwnProps} Props */
+
+/**
+ * @param {import('@wp-g2/create-styles').ViewOwnProps<Props, 'div'>} props
+ */
+export function useBaseField( props ) {
+	const {
+		className,
+		error = false,
+		isClickable = false,
```

Comment:
I know that the README states that `isClickable` only adds a visual change on hover, but it still sounds like a bit of a misleading prop name — I initially assumed that, when `isClickable=false`, the used wouldn't be able to click on the component at all. WDYT?

Reply:
That behavior would conflict with `disabled` so I don't think `isClickable=false` should do that. But I agree it's maybe misleading. Maybe we should just add a `cursorPointer` prop instead? 🤔 I'm not sure what the use case for a "clickable" input field is to be honest.

Reply:
We could either change it to something like `cursorOnHover: CSSProperty['cursor']`, or (my preferred option) just remove the prop altogether — the idea being that this piece of functionality would basically just be a proxy for a CSS prop

WDYT?

Reply:
> the idea being that this piece of functionality would basically just be a proxy for a CSS prop

If we had the `css` prop I would agree with you 100%. But in this case I agree mostly because I don't personally see the use case for this prop at all.

Reply:
Agreed — let's remove the `isClickable` prop